Go Client/Go Rules.cpp

```diff
@@ -58,31 +61,32 @@ uint8_t Rules::CheckLibertiesOfAGroup(uint8_t x, uint8_t y)
 	CheckLibertiesOfAStone(x, y);
 	if (x < currentBoard.size() - 1)
```

Comment:
+1s and -1s are confusing things aren't they? :) comments help explain why you're doing it..